doc/releases/release-notes-2.4.rst

```diff
@@ -409,6 +416,11 @@ Networking
 * Fixed LWM2M Security and Server object instance matching.
 * Fixed LWM2M updating lifetime on Register Update event.
 * Fixed MQTT double CONNACK event notification on server reject.
+* PPP: Added initial support for PAP authentication.
+* PPP: Automatic select CONFIG_RING_BUFFER as it is required.
+* PPP: Refactored option negotiation.
+* PPP: Fixed a race condition when ppp_fsm_open() was called in CLOSED state.
```

Comment:
This could be placed after other fixed PPP stuff above. So something like
```
* Fixed a race PPP when ppp_fsm_open() was called in CLOSED state.
```